.github/actions/test-install-enable/install_and_enable.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+echo "Installing Wazuh $2."
+
+source /etc/os-release
+if [ "$ID" = "centos" ] && [ "$VERSION_ID" = "8" ]; then
+    find /etc/yum.repos.d/ -type f -exec sed -i 's/mirrorlist/#mirrorlist/g' {} \;
+    find /etc/yum.repos.d/ -type f -exec sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' {} \;
+fi
+
+if [ -n "$(command -v yum)" ]; then
+    sys_type="yum"
+elif [ -n "$(command -v apt-get)" ]; then
+    sys_type="apt-get"
+    apt-get update
+    apt-get install -y systemd
+else
+    common_logger -e "Couldn'd find type of system"
```

Comment:
Please change `Couldn'd` to `Couldn't`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2053/commits/24734fb12d34b2b331876e697f44ebe4ad25e0af